deb/openmediavault/workbench/src/app/core/services/dashboard-widget-config.service.ts

```diff
@@ -76,23 +85,74 @@ export class DashboardWidgetConfigService {
   }
 
   /**
-   * Get the identifiers of all enabled widgets of the current user.
-   *
-   * @return Returns a list of widget identifiers (UUID).
+   * Load the user's dashboard widget configuration.
    */
-  public getEnabled(): Array<string> {
-    const value = this.userLocalStorageService.get('dashboard_widgets', '[]');
-    const result: Array<string> = JSON.parse(value);
-    return result;
+  public getEnabled(): Observable<DashboardUserConfig> {
+    console.warn('calling get enabled');
+    const emptyConfig: DashboardUserConfig = { widgets: [] };
+    return this.rpcService
+      .request('Dashboard', 'getDashboardSetting', { deviceType: this.platformDeviceType })
+      .pipe(
+        take(1),
+        catchError((error) => {
+          if (_.isFunction(error.preventDefault)) {
+            error.preventDefault();
+          }
+          return of(emptyConfig);
+        }),
+        mergeMap((userWidgetConfig: DashboardUserConfig) => {
```

Comment:
Complexity here is to allow for backwards compatibility.
Will take whatever is stored in local storage, push it to the server, and then delete it from local storage